service-commands/src/libs.js

```diff
@@ -384,12 +385,12 @@ function LibsWrapper (walletIndex = 0) {
       this.getWalletAddress()
     )
 
-    if (!users.length) {
-      throw new Error('No users!')
+    // If a user is found, set this libs instance's userId
+    if (users.length > 0 && !this.userId) {
```

Comment:
why was this changed?

Reply:
i think we should still throw if this condition is met right? `if(!users.length)`?

Reply:
when I was wrapping the APIs, I needed the libsWrapper to not throw on non-actual errors. Now that we're wrapping the executeOne/executeAll with a try/catch, this change isn't necessary anymore. will update

Reply:
actually, now that we wrap the executeOne/executeAll into a try/catch, any non-actual error throws will be a false positive in the test erroring. going to handle all non-actual error throws in this file

Reply:
scratch the last comment about handling all non-actual errors. will revert